### PR TITLE
Build Queue Support

### PR DESCRIPTION
**Pull request by hugener**
Added support for getting the build queue by build type locator and project locator.

The REST API is documented here:
https://confluence.jetbrains.com/display/TCD10/REST+API#RESTAPI-queuedBuildsQueuedBuilds

### CODE REVIEW

File: src/Tests/IntegrationTests/SampleBuildQueueUsage.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+using System.Configuration;
+using NUnit.Framework;
+using TeamCitySharp.Locators;
+
+namespace TeamCitySharp.IntegrationTests
+{
+    [TestFixture]
```
Comment (mavezeau) [inline src/Tests/IntegrationTests/SampleBuildQueueUsage.cs]: thanks for your test :)

File: src/TeamCitySharp/ActionTypes/BuildQueue.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+using TeamCitySharp.Connection;
+using TeamCitySharp.DomainEntities;
+using TeamCitySharp.Locators;
+
```
Comment (mavezeau) [inline src/TeamCitySharp/ActionTypes/BuildQueue.cs:5]: Normally, I use 2 space not 4 space... I will change this before to publish a new version

**Review by mavezeau — APPROVED** [review]

File: src/TeamCitySharp/ActionTypes/BuildQueue.cs (hunk 2 above)
Comment (hugener) [reply src/TeamCitySharp/ActionTypes/BuildQueue.cs:5]: Great thanks, can you give a hint as to when the nuget package will be updated?

Comment (mavezeau) [reply src/TeamCitySharp/ActionTypes/BuildQueue.cs:5]: Very soon!!

Comment (mavezeau) [reply src/TeamCitySharp/ActionTypes/BuildQueue.cs:5]: done. We need to wait that nuget.org authorize the package

https://www.nuget.org/packages/TeamCitySharp-forked-mavezeau/1.0.58

Comment (hugener) [reply src/TeamCitySharp/ActionTypes/BuildQueue.cs:5]: Cool, thanks!

**Pull request merged**